TextureHolder.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include<SFML/Graphics.hpp>
+class TextureHolder
```

Comment:
## "Resource manager" pattern...
...którego nie znam i nie wiedziałem że istnieje.

Jeżeli napotkasz przy pisaniu kodu coś, na co nie masz pomysłu, to warto to wygooglować.
Zarządzanie assetami jest potrzebne przy każdym silniku - mnóstwo ludzi potrzebowało napisać coś takiego.

Głównym problemem twojego podejścia jest to, że to jest dosłownie texture holder - tablica tekstór z 'fancy' indeksami.
Nie daje to możliwości rozszerzenia tej struktury:

1. dla stu tekstur jej implementacja byłaby absurdalna.
2. nazwy tekstór będą raczej przechowywane w jakimś pliku opisującym assety (jakie są i gdzie)